qiskit_sphinx_theme/__init__.py

```diff
@@ -25,7 +26,9 @@ def _get_theme_absolute_path(folder_name: str) -> str:
     return str(path.resolve())
 
 
-def remove_thebe_if_not_needed(app, pagename, templatename, context, doctree):
+def remove_thebe_if_not_needed(
+    app: Any, pagename: Any, templatename: Any, context: Any, doctree: Any
```

Comment:
We know what `app` is. Look at how we do this in other repositories:

https://github.com/Qiskit/qiskit_sphinx_theme/blob/67fd6ac2658803325b2536460b511c8eb349d214/qiskit_sphinx_theme/translations.py#L16-L20

It would also be good to figure out what the other values are and avoid using `Any`. Try using `reveal_type`: https://adamj.eu/tech/2021/05/14/python-type-hints-how-to-debug-types-with-reveal-type/

This is a useful skill for the Terra project.